docs_versioned_docs/version-ros2humble/ros/config/live.mdx

```diff
@@ -8,30 +8,46 @@ toc_max_heading_level: 4
 
 Users familiar with Clearpath's robots, know we provide desktop packages that facilitate the viewing of robot models in RViz without needing to have a real robot. In ROS 2, the `view_model.launch.py` node will check and update the model and display the new changes in RViz.
 
-This functionality requires a desktop manager to be installed; in other words, these commands will not work on robot computers since these run Ubuntu 22.04 server rather than desktop.
+This functionality requires a desktop manager to be installed; in other words, these commands will not work on standard robot computers since these run Ubuntu 22.04 server rather than desktop.
 
 # Installation
+
 Install the desktop packages using `apt`. Make sure to have [added Clearpath packages](https://packages.clearpathrobotics.com/) to the source list before installation.
+
 ```bash
 sudo apt install ros-humble-clearpath-desktop ros-humble-clearpath-config-live
 ```
 
 # Setup
+
 Before running the node, we will need to setup the working directory.
-First, create the directory `/etc/clearpath`.
-Then, copy your `robot.yaml` to `/etc/clearpath/robot.yaml`.
+
+Our standard working directory is: `/etc/clearpath`. However, you can choose whatever directory you'd like. We will refer to this directory as the **setup_path**.
```

Comment:
`/etc/clearpath` is our standard working directory on the robot but in the "remote computer setup" (which we direct people to use for setting up their PC for simulation) we indicate `~/clearpath` as the standard. Maybe we should reference that here to say "Our standard working directory on robots is:  ... and our standard working directory on remote PC's and desktops is: ... or something similar. Or as an alternative below indicate that if they have setup their PC as per those instructions, the setup path is `$HOME/clearpath/`